src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,20 @@
+package racingcar.domain
+
+class Car(private val moveStrategy: MoveStrategy, private val moveDepth: Int = 1) {
+
+    private var distance: Int = 0
```

Comment:
setter는 막고, 현재 distance 값을 가져오는 부분을 허용하고 싶다면 아래와 같이 정의가 가능하겠네요.

```suggestion
    var distance: Int = 0
        private set
```

Reply:
`private set` 은 setter를 막고 public getter를 제공하는군요.

https://github.com/next-step/kotlin-racingcar/blob/d9db06259bb1edb308eca0874e628e9fd4f167ed/src/main/kotlin/racingcar/domain/Car.kt#L17-L19

위 메서드의 역할이 getter와 다를바 없다고 생각은 해요.

하지만, `private set` 사용 시 자동차의 거리를 가져오기 위해 `car.distance` 라는 코드가 개인적으로 낯설게 느껴지는데요. 
`car.distance`는 `car` 객체로부터 데이터를 꺼내는 것 처럼 보여져서 그런 것 같아요.

리뷰어님은 이럴 때 `private set` 을 사용하시나요? 아니면 위와 같이 의식적으로 getter를 피하실지 궁금합니다.

(따로 코드에 반영하지는 않았어요 🙏)